pkg/storage/store.go

```diff
@@ -83,6 +83,8 @@ type ChunkStoreConfig struct {
 // RegisterFlags adds the flags required to configure this flag set.
 func (cfg *ChunkStoreConfig) RegisterFlags(f *flag.FlagSet) {
 	cfg.StoreConfig.RegisterFlags(f)
+	_ = cfg.MaxLookBackPeriod.Set("0s")
+	f.Var(&cfg.MaxLookBackPeriod, "store.chunks-cache.max-look-back-period", "This flag is deprecated. Use -querier.max-query-lookback instead.")
```

Comment:
```suggestion
	f.Var(&cfg.MaxLookBackPeriod, "store.max-look-back-period", "This flag is deprecated. Use -querier.max-query-lookback instead.")
```
We should omit the `chunks-cache` prefix here:
https://github.com/grafana/loki/blob/main/pkg/storage/store.go#L91